src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -89,9 +117,18 @@
  * <pre>
  * try {
  *   throw new RuntimeException();
+ * } catch (RuntimeException expected) {
+ * } // violation
+ * try {
+ *   throw new RuntimeException();
+ * } catch (RuntimeException ignore) {
+ *   // no handling
+ * } // violation
+ * try {
+ *   throw new RuntimeException();
  * } catch (RuntimeException ex) {
- *   //This is expected
- * }
+ *   // This is expected
```

Comment:
move it to first code example.

Reply:
@romani I haven't added `// This is expected` on the second example too.I did that because users can understand there is a violation because the variable is `ex`. Do I need to do it over here too?

Reply:
add to all examples

Reply:
Okayy